downloadservice/app.py

```diff
@@ -360,8 +364,25 @@ def upload(user, path):
     if '..' in path:
         return "Error: path cannot contain '..'", 400
 
+    # check if upload folder is accessible
+    potential_folders = app.config['CTADS_UPSTREAM_UPLOAD_FOLDERS']
+    selected_base_folder = None
+    for base_folder in potential_folders:
+        try:
+            _, status_code = list(path=urljoin_multipart(base_folder, 'users'))
```

Comment:
```suggestion
            _, status_code = _list(path=urljoin_multipart(base_folder, 'users'))
```

Reply:
we should really rename the `list` which is reserved

Reply:
Renamed, but not with with an underscore as this as the meaning of internal use in python.